theano/gpuarray/tests/test_linalg.py

```diff
@@ -216,7 +241,7 @@ def test_gpu_matrix_inverse(self):
         # Copied from theano.tensor.tests.test_basic.rand.
         A_val = test_rng.rand(N, N).astype('float32') * 2 - 1
         A_val_inv = fn(A_val)
-        utt.assert_allclose(np.eye(N), np.dot(A_val_inv, A_val), atol=5e-3)
+        utt.assert_allclose(np.dot(A_val_inv, A_val), np.eye(N), atol=1e-2)
```

Comment:
In fact, the right order for assert_allclose args is `expected, value`. So, `np.eye(N)` should go first, as it's the expected value !

(we will have to fix it in whole Theano code, but later!)